apps/meteor/app/file-upload/server/config/GoogleStorage.js

```diff
@@ -77,6 +78,8 @@ const configure = _.debounce(function () {
 		return;
 	}
 
+	process.env.GCLOUD_PROJECT = projectId;
```

Comment:
curious, why is this needed here?

Reply:
this is necessary according with the issues of github about gcloud on 4.8.0